stat/stat.go

```diff
@@ -28,6 +28,7 @@ const (
 	MethodObjectSearch
 	MethodContainerAnnounceUsedSpace
 	MethodSyncContainerWithNetwork
+	// methodLast is no a valid method name, it's a system anchor for tests, etc.
```

Comment:
This comment is added in a separate commit from `methodLast` definition.

Reply:
Fixed